exchanges/gateio/gateio_types.go

```diff
@@ -1301,7 +1301,7 @@ type CrossMarginBalance struct {
 	BorrowedNet         string       `json:"borrowed_net"`
 	TotalNetAssetInUSDT string       `json:"net"`
 	PositionLeverage    string       `json:"leverage"`
-	Risk                string       `json:"risk"` // Risk rate. When it belows 110%, liquidation will be triggered. Calculation formula: total / (borrowed+interest)
+	Risk                string       `json:"risk"` // Risk percentage; Liquidation is triggered when this falls below required margin. Calculation: total / (borrowed+interest)
```

Comment:
Everything looks good.

But:
> Grabs the latest PR suggestion for the fix to gateio's risk type

This is just a comment change. Was an actual fix supposed to be here?

Reply:
Fixes the codespell failure about the word `belows`